toolip/fastapi_tools/config.py

```diff
@@ -2,8 +2,8 @@
 
 
 class Configuration(BaseSettings):
-    doc_username: str
-    doc_password: str
+    doc_username: str = ""
+    doc_password: str = ""
```

Comment:
Hummmm ... not sure this is the right way.
How about that?

```suggestion
    doc_username: Optional[str] = None
    doc_password: Optional[str] = None
    
    @property
    def doc_auth_is_on(self):
        return self.doc_username is not None and self.doc_password is not None
```

We also need to [validate](https://pydantic-docs.helpmanual.io/usage/validators/) all in the interest of not exposing docs accidentally in production so that both doc_username and doc_password are set together and not just one of them.

Reply:
I forgot to mention that the auth should be turned off for docs if doc_auth_is_on == False